packages/core/useDraggable/component.ts

```diff
@@ -36,6 +36,7 @@ export const UseDraggable = /* #__PURE__ */ defineComponent<UseDraggableProps>({
     'disabled',
     'buttons',
     'containerElement',
+    'capture',
   ] as unknown as undefined,
```

Comment:
we can actually drop this cast now i think

no action needed. feel free to do it in this PR, or don't and we can do it in a follow-up